src/exports.jl

```diff
@@ -14,6 +9,25 @@
 
 @reexport using Extents
 @reexport using GeoInterface: crs
+@reexport using DataFrames
+@reexport using GeometryOps:
+    intersects,
```

Comment:
overlaps with Base.intersects, Base.equals, etc...what is the plan there?

Reply:
Yeah I think predicates have to be internal everywhere

Reply:
The responsibility lies with GeometryOps to know what to export

But GeoDataframes can import or using these predicates without exporting so they are accessible via GDF.*